src/helpers/Environment.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Access `process.env` in an environment helper
+ * Usage: `EnvHelper.env`
+ * - Other static methods can be added as needed per
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static
+ */
+export class EnvHelper {
+  /**
+   * @returns `process.env`
+   */
+  static env = process.env;
+  static alchemyTestnetURI = `https://eth-rinkeby.alchemyapi.io/v2/${EnvHelper.env.REACT_APP_TESTNET_ALCHEMY}`;
+
+  static whitespaceRegex = /\s+/;
+
+  static isNotEmpty(envVariable: string) {
+    if (envVariable.length > 10) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+  /**
+   * in development environment will return the `ethers` community api key so that devs don't need to add elements to their .env
+   * @returns Array of Alchemy API URIs or empty set
+   */
+  static getAlchemyAPIKeyList() {
+    let ALCHEMY_ID_LIST: string[];
+
+    // split the provided API keys on whitespace
+    if (
+      EnvHelper.env.NODE_ENV !== "development" &&
+      EnvHelper.env.REACT_APP_ALCHEMY_IDS &&
+      EnvHelper.isNotEmpty(EnvHelper.env.REACT_APP_ALCHEMY_IDS)
+    ) {
+      ALCHEMY_ID_LIST = EnvHelper.env.REACT_APP_ALCHEMY_IDS.split(EnvHelper.whitespaceRegex);
+    } else {
+      // this is the ethers common API key, suitable for testing, not prod
+      ALCHEMY_ID_LIST = ["_gg7wSSi0KMBsdKnGVfHDueq6xMB9EkC"];
+    }
```

Comment:
@lolchocotaco line 39 -> in development environment ALCHEMY_ID_LIST will automatically have the `ethers` common API key added. So even if a dev has a COMPLETELY BLANK .env file... the app will still run properly.